### PR TITLE
Adding Laravel demo

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR adds a Laravel app demo - OctoFacts - to the php demos folder. I am using this demo for a "Getting Started" guide for the Laravel image.

I was a bit unsure if this is the best place for this demo, since it is a larger application with some images (that are shown randomly alongside a quote / fact about Octopuses). 

Let me know if it's best to have a separate repo for this - my only concern is making the repository bloated from all the framework shenanigans + images.

### CODE REVIEW

**Comment by amouat** [comment]
sorry, just realised I should have reviewed this

**Comment by amouat** [comment]
I think it's fine to be in this repo, it's in a folder so no-one will notice.

**Review by amouat — APPROVED** [review]
This looks fine, but I have to say I have no idea about PHP!

A few things that don't need to hold things up:

 - there's a lot of dotfiles, especially .gitignore; do we need them all?
 - should there be a Dockerfile? How do I test it with Chainguard Images?
 - if there's a blog/article to go with this, can we add a link to the README? (I'm guessing it's not published yet?

File: php/octo-facts/app/Models/User.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Models;
+
+// use Illuminate\Contracts\Auth\MustVerifyEmail;
```
[the remark this answers is not in the document]
Comment (erikaheidi) [reply php/octo-facts/app/Models/User.php:5]: Hm it's not really needed, this is part of the initial skeleton of the application. It is pretty standard as an option that users might enable afterwards

**Comment by erikaheidi** [comment]
> This looks fine, but I have to say I have no idea about PHP!
> 
> A few things that don't need to hold things up:
> 
> * there's a lot of dotfiles, especially .gitignore; do we need them all?
> * should there be a Dockerfile? How do I test it with Chainguard Images?
> * if there's a blog/article to go with this, can we add a link to the README? (I'm guessing it's not published yet?

yes! all this. well, the .gitignore files are necessary to build the default folder structure. I wanted to keep here the demo only (so users clone it from the tutorial), and have the Dockerfile / Docker Compose file in the tutorial only.  I thought this would keep the demo agnostic for any different use cases. About the README, yes I plan to add a link to the guide when it's published! I am getting it finished soon-ish (hopefully today).

**Pull request merged**